src/main/resources/example.json

```diff
@@ -0,0 +1,26 @@
+{

```

Comment:
Should these example jsons be in the main directory? 
Maybe they fit in /src? Or some examples or docs directory?